### DISCUSSION TITLE
How to use S3 benchmarks

### DISCUSSION THREAD

**Discussion by jovans2** [vhive-serverless/vHive#410] [Q&A]
Hello,

It is not possible to run pre-existing benchmarks that use S3 storage: video_processing and lr_training

Error: 
"Failed to get/pull image: failed to resolve reference "ghcr.io/ease-lab/video_processing_s3:var_workload": failed to authorize: failed to fetch anonymous token: unexpected status: 403 Forbidden"

How to run these apps?
Thank you!

**Comment by ustiugov** [comment]
hi @jovans2, we dropped support for the benchmarks that use MinIO object store. For reproducibility purposes, you can check out the branch from this [PR](https://github.com/ease-lab/vhive/pull/74/files); this is the code that we ran for the ASPLOS'21 work. 

We have been working on a more sophisticated workload [suite](https://github.com/ease-lab/vSwarm) where we mostly used AWS S3 in place of MinIO. This suite is in beta now but it's usable. 

Could you please describe what you are trying to do and which research goals you are pursuing?
